### ISSUE TITLE
Docker extension v1.29.2 in WSL broken

### ISSUE THREAD

**Issue by danikane** [microsoft/vscode-docker#4364]
**wsl -v**
```
WSL version: 2.2.4.0
Kernel version: 5.15.153.1-2
WSLg version: 1.0.61
MSRDC version: 1.2.5326
Direct3D version: 1.611.1-81528511
DXCore version: 10.0.26091.1-240325-1447.ge-release
Windows version: 10.0.19045.4412
```

**docker info**
```
Client: Docker Engine - Community
 Version:    27.1.2
 Context:    default
 Debug Mode: false
 Plugins:
  buildx: Docker Buildx (Docker Inc.)
    Version:  v0.16.2
    Path:     /usr/libexec/docker/cli-plugins/docker-buildx
  compose: Docker Compose (Docker Inc.)
    Version:  v2.29.1
    Path:     /usr/libexec/docker/cli-plugins/docker-compose

Server:
 Containers: 4
  Running: 2
  Paused: 0
  Stopped: 2
 Images: 9
 Server Version: 27.1.2
 Storage Driver: overlay2
  Backing Filesystem: extfs
  Supports d_type: true
  Using metacopy: false
  Native Overlay Diff: true
  userxattr: false
 Logging Driver: json-file
 Cgroup Driver: cgroupfs
 Cgroup Version: 1
 Plugins:
  Volume: local
  Network: bridge host ipvlan macvlan null overlay
  Log: awslogs fluentd gcplogs gelf journald json-file local splunk syslog
 Swarm: active
  NodeID: xzgt3w5skpgmyy2ad1mhcfq0n
  Is Manager: true
  ClusterID: ql78y8dx4s6f3qmxfgc87t64p
  Managers: 1
  Nodes: 1
  Default Address Pool: 10.0.0.0/8  
  SubnetSize: 24
  Data Path Port: 4789
  Orchestration:
   Task History Retention Limit: 1
  Raft:
   Snapshot Interval: 10000
   Number of Old Snapshots to Retain: 0
   Heartbeat Tick: 1
   Election Tick: 10
  Dispatcher:
   Heartbeat Period: 5 seconds
  CA Configuration:
   Expiry Duration: 3 months
   Force Rotate: 0
  Autolock Managers: false
  Root Rotation In Progress: false
  Node Address: 172.19.52.56
  Manager Addresses:
   172.19.52.56:2377
 Runtimes: io.containerd.runc.v2 runc
 Default Runtime: runc
 Init Binary: docker-init
 containerd version: 8fc6bcff51318944179630522a095cc9dbf9f353
 runc version: v1.1.13-0-g58aa920
 init version: de40ad0
 Security Options:
  seccomp
   Profile: builtin
 Kernel Version: 5.15.153.1-microsoft-standard-WSL2
 Operating System: Debian GNU/Linux 12 (bookworm)
 OSType: linux
 Architecture: x86_64
 CPUs: 12
 Total Memory: 15.59GiB
 Name: TOPDESK
 ID: 4787be6b-0a43-4a5d-983f-f91eda35c30e
 Docker Root Dir: /var/lib/docker
 Debug Mode: false
 Username: danieldconex
 Experimental: false
 Insecure Registries:
  127.0.0.0/8
 Live Restore Enabled: false
```

v1.29.1
--------
![image](https://github.com/user-attachments/assets/1144bf71-6990-4d01-a727-cbfe12206087)
![image](https://github.com/user-attachments/assets/16557f25-f521-4008-8f5b-642006d60b74)

All good!

v1.29.2
-------
![image](https://github.com/user-attachments/assets/0e8ff263-1992-480d-ab48-5aad1076283b)

Not good. :)
Thanks for your attention!

Revert to the previous version is possible, so this is great.

**Comment by bwateratmsft** [comment]
It looks like the extension didn't install properly. That or it's not activating. Can you check the Output > Extension Host output window to see if there is anything interesting?

**Comment by danikane** [comment]
Well I'll be damned 😄 
This time I updated it it worked!
I did do several cycles or uninstall/install before with the same result, but now it works.
Maybe it's worth mentioning I subscribed to the pre-release channels of these extensions, but they haven't been updated soon whereas latest Docker extension is less than a day old:
![image](https://github.com/user-attachments/assets/6a767858-dd10-4e3e-b20f-073d98015577)
 
![image](https://github.com/user-attachments/assets/4640b036-71b7-44d6-924f-6389018b3e9e)
![image](https://github.com/user-attachments/assets/5484d19e-07ba-46fe-9e45-43dbe151b172)

Should I close the issue then?
